app/assets/stylesheets/content/_advanced_filters.sass

```diff
@@ -29,88 +29,61 @@
 .advanced-filters--container
   @extend %filters--container
   padding: rem-calc(10px)
-  max-height: 50vh
-  overflow-y: auto
-  position: relative
+  margin: 0
+  overflow: hidden
 
   &.collapsed
     display: none
 
   .advanced-filters--close
-    position: absolute
-    top: 0.75rem
-    right: 0.75rem
+    // Flex would be a nicer solution, but will not work here (for unknown reasons)
+    float: right
     width: 1rem
     z-index: 2
 
-@mixin advanced-filters--sizing()
-  @include grid-block()
-
-  @include breakpoint(medium)
-    @include grid-size(12)
-
-  @include breakpoint(large)
-    @include grid-size(10)
-
-  @include breakpoint(xlarge)
-    @include grid-size(8)
-
-  @include breakpoint(xxlarge)
-    @include grid-size(6)
-
-  @include breakpoint(xxxlarge)
-    @include grid-size(5)
-
-.advanced-filters--filters
+.advanced-filters--filters,
+.simple-filters--filters
```

Comment:
The file is named `advanced-filters`, so please do not mix the styles here. This will only make it harder in the future. If they share the same style let the advanced filters inherit the styles from the simple filters.